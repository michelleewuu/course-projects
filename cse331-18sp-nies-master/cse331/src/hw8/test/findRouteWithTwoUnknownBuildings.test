# Verify that the program prints 
# "Unknown building: " + buildingName 
# twice when both starting building and 
# ending building are unknown.
r
HAG
MCH

# terminate the program
q