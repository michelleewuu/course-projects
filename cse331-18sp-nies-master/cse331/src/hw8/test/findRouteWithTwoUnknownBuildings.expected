Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program prints 
# "Unknown building: " + buildingName 
# twice when both starting building and 
# ending building are unknown.
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: HAG
Unknown building: MCH

Enter an option ('m' to see the menu): 
# terminate the program